editor/src/templates/image-dnd.spec.browser2.tsx

```diff
@@ -233,17 +233,17 @@ describe('image dnd', () => {
 
     const imageDragHandle = editor.renderedDOM.getByTestId('file-image-drag-handle')
     const dragHandleBounds = imageDragHandle.getBoundingClientRect()
-    const handleCenter = slightlyOffsetPointBecauseVeryWeirdIssue({
+    const handleCenter = {
       x: dragHandleBounds.x + dragHandleBounds.width / 2,
       y: dragHandleBounds.y + dragHandleBounds.height / 2,
-    })
+    }
 
     const target = editor.renderedDOM.getByTestId('scene')
     const targetBounds = target.getBoundingClientRect()
 
     const endPoint = {
-      x: targetBounds.x + targetBounds.width / 2,
-      y: targetBounds.y + targetBounds.height / 2,
+      x: (targetBounds.x + targetBounds.width / 2) ^ 0,
```

Comment:
I think Math.round or Math.floor would be much more readable here.

Reply:
fixed